pkg/action/archive.go

```diff
@@ -51,7 +51,7 @@ func copyArchive(ctx context.Context, src string, dst string) error {
 func tempDir(ctx context.Context, p string) (string, error) {
 	logger := clog.FromContext(ctx).With("path", p)
 	logger.Info("creating temp dir")
-	tmpDir, err := os.MkdirTemp("", fmt.Sprintf("bincapz-%s", filepath.Base(p)))
```

Comment:
This was a fun side-effect of #167. The workaround would be `--ignore-self=false` but that's a lot of effort when this should Just Work™️.